_posts/fr/newsletters/2024-08-30-newsletter.md

```diff
@@ -0,0 +1,145 @@
+
+Changements récents notables dans [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], et [BINANAs][binana repo]._
+
+- [LDK #3263][] simplifie la manière dont il gère les réponses aux [messages onion][topic onion messages]
+  en supprimant le paramètre de type de message de la structure `ResponseInstruction`,
+  et en introduisant un nouveau enum `MessageSendInstructions` basé sur
+  la mise à jour de `ResponseInstruction`, qui peut gérer à la fois les chemins de réponse
+  [aveuglés][topic rv routing] et non aveuglés. La méthode `send_onion_message` utilise maintenant
+  `MessageSendInstructions`, permettant aux utilisateurs de spécifier des chemins de réponse sans
+  avoir à résoudre eux-mêmes le routage. Une nouvelle option, `MessageSendInstructions::ForReply`,
+  permet aux gestionnaires de messages d'envoyer des réponses plus tard sans créer de dépendances
+  circulaires dans le code. Voir le Bulletin [#303][news303 onion].
+
+- [LDK #3247][] déprécie la méthode `AvailableBalances::balance_msat` au profit de la méthode
+  `ChannelMonitor::get_claimable_balances`, qui offre une approche plus directe et précise pour
+  obtenir le solde d'un canal. La logique de la méthode dépréciée est désormais obsolète car elle
+  était initialement conçue pour gérer les problèmes de sous-flux potentiels lorsque les soldes
+  incluaient des HTLCs en attente (ceux qui pourraient être inversés plus tard).
+
+- [BDK #1569][] ajoute la paquet `bdk_core` et y déplace certains types de `bdk_chain` : `BlockId`,
+  `ConfirmationBlockTime`, `CheckPoint`, `CheckPointIter`, `tx_graph::Update` et `spk_client`. Les
+  sources de chaînes `bdk_esplora`, `bdk_electrum` et `bdk_bitcoind_rpc` ont été modifiées pour
+  dépendre uniquement de `bdk_core`. Ces changements ont été effectués pour permettre un remaniement
+  plus rapide sur `bdk_chain`.
+
+{% assign four_days_after_posting = page.date | date: "%s" | plus: 345600 | date: "%Y-%m-%d 14:30" %}
+{% include snippets/recap-ad.md when=four_days_after_posting %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="3263,3247,1569" %}
+[Core Lightning 24.08rc2]: https://github.com/ElementsProject/lightning/releases/tag/v24.08rc2
+[LND v0.18.3-beta.rc1]: https://github.com/lightningnetwork/lnd/releases/tag/v0.18.3-beta.rc1
+[BDK 1.0.0-beta.2]: https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.2
+[bitcoin core 28.0rc1]: https://bitcoincore.org/bin/bitcoin-core-28.0/
+[news315 oblivious]: /en/newsletters/2024/08/09/#block-withholding-attacks-and-potential-solutions
```

Comment:
Once we get changes to fr315 merged we can update this link

Reply:
Done